tests/Doctrine/Migrations/Tests/Version/VersionTest.php

```diff
@@ -178,10 +186,12 @@ public function testAddSql() : void
             VersionDummy::class
         );
 
-        self::assertNull($version->addSql('SELECT * FROM foo'));
-        self::assertNull($version->addSql('SELECT * FROM foo'));
-        self::assertNull($version->addSql('SELECT * FROM foo WHERE id = ?', [1]));
-        self::assertNull($version->addSql('SELECT * FROM foo WHERE id = ?', [1], [PDO::PARAM_INT]));
+        $version->addSql('SELECT * FROM foo');
+        $version->addSql('SELECT * FROM foo');
+        $version->addSql('SELECT * FROM foo WHERE id = ?', [1]);
+        $version->addSql('SELECT * FROM foo WHERE id = ?', [1], [PDO::PARAM_INT]);
+
+        self::markTestIncomplete('Does not perform any real assertion.');
```

Comment:
For these cases we should use `$this->addToAssertionCount(1)`, more info: https://github.com/sebastianbergmann/phpunit/pull/3348